content/en/about/case-studies/bol.com/index.md

```diff
@@ -0,0 +1,60 @@
+
+## Challenge
+
+As they began to migrate operations to the cloud, bol.com faced inevitable growing pains. They began moving applications to a Kubernetes cluster, adding more and more pods over time. It seemed like the cluster address space had plenty of room. Unfortunately, scaling for demand was quickly an issue. They initially configured the cluster with a service CIDR with space for about 1,000 addresses, but just one year later, they were already at 80% capacity.
+
+Roland Kool is one of the system engineers on the bol.com team that addressed this issue. Faced with the knowledge that available IP address space in their Kubernetes cluster would not keep up with growing needs, the team needed a solution that would enable overflow into additional clusters. In addition, this new multi-cluster Kubernetes deployment would bring new networking challenges, as applications would need a new approach to service discovery, load balancing, and secure communication.
+
+## Solution: Multiple Clusters with a Service Mesh
+
+The solution seemed to be introducing additional clusters, but they ran into issues with security requirements and network policies that secured traffic between services.
+
+This challenge was exacerbated by the need to protect personally identifying information (PII). Due to European regulations such as GDPR, every service touching PII needs to be identified and access needs to be strictly controlled.
+
+Since network policies are cluster local, they don’t work across cluster boundaries. All of those per-cluster network policies got messy fast. They needed a solution that would allow them to apply security at a higher layer.
+
+Istio's [multi-cluster deployment model](/docs/ops/deployment/deployment-models/#multiple-clusters) ended up being the perfect solution. [Authorization policies](/docs/reference/config/security/authorization-policy/) could be used to securely allow workloads from different clusters to talk to each other. With Istio, Kool's team was able to move away from OSI layer 3 or 4 network policies to [authz policies](/docs/tasks/security/authorization/authz-http/) implemented in layer 7. This move was made possible by Istio's strong identity support, service-to-service authentication, and security with mutual TLS (mTLS).
+
+These changes gave bol.com the ability to scale by adding new Kubernetes clusters while maintaining service discovery, load balancing, and required security policies.
+
+## Why Istio?
+
+When bol.com initially began migrating to Kubernetes, Istio was just in version 0.2. It did not seem to be ready for production, so they went forward without Istio. They first started to look seriously at Istio around version 1.0, but they ran into too many issues with deployment and implementation. Without an urgent use case, they tabled the idea.
```

Comment:
Reads weird to me.  was "only at" rather than was "just in"?

Reply:
Do you want this edit made? Is this the last one?

Reply:
Yes, assuming you're happy with my suggestion.